use-case/financial/index.md

```diff
@@ -53,13 +53,13 @@ Accept: application/json+fhir
 ```
 
 ##### On *periodic-update*
-    GET /Coverage?patient={% raw %}{{patientId}}{% endraw %}&_lastUpdated=gt{% raw %}{{lastCheck}}{% endraw %}
+    GET /Coverage?beneficiary={% raw %}{{patientId}}{% endraw %}&_lastUpdated=gt{% raw %}{{lastCheck}}{% endraw %}
```

Comment:
Good catch -- I think there *should* be a param called `patient` (and raised this point [in discussion](https://chat.fhir.org/#narrow/stream/implementers/topic/Searching.20Coverage.20by.20patient)) but in STU3 there isn't. Thanks!